app/docs/how-to/streamline-curbside.md

```diff
@@ -0,0 +1,255 @@
+attrs:
+  successMessage: Thank You!
+order:
+  - order_type
+  - name
+  - car
+required:
+  - name
+  - car
+  - order_type
+properties:
+  car:
+    type: string
+    attrs:
+      placeholder: E.g. White Honda Accord
+    title: Please give us some details about your car
+  name:
+    type: string
+    attrs:
+      placeholder: E.g. John
+    title: What is the name on the order?
+  order_type:
+    enum:
+      - By Phone
+      - Online
+      - GrubHub
+      - Other
+    type: string
+    attrs:
+      type: radio
+    title: How did you place your order?
+```
+
+This YAML document defines a simple schema with **car**, **name** and **order_type** fields, and it can be easily tailored to meet the needs of your business. For instance, you can easily modify titles of these fields by changing **title** attribute under each individual field.
+
+To apply this document to our newely created schema, scroll down to **Definition (YAML)** section of the form, and copy/paste the YAML code above replacing the original content.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-schema-yaml.png" width="75%">
+</p>
+
+Once done, you can click the **Preview** tab to inspect the contents of the schema form.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-schema-preview.png" width="75%">
+</p>
+
+#### Step 5: Save and test your touchpoint
+
+If you are happy with the name, description and schema, click **Create touchpoint** button to save the changes, and you will be presented with a QR code and a URL for your new touchpoint.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/touchpoint-qr.png" width="75%">
+</p>
+
+Scan the code with a camera app on your phone (or click the link) to make sure that the your new curbside pickup touchpoint looks and works just like you defined. You can always go back and tweak the name, description and schema if needed.
+
+
+### Setting up your Trello board
+
+Next, we are going to configure your Trello board. In order to keep track of arriving customers and delivery status we are going to create a Trello board with two lists:
+* *Waiting Curbside Customers*
+* *Delivered Curbside Customers*
+
+Routegy will automatically add a new Trello card to the first list every time a customer interacts with the touchpoint that you defined earlier. Then, you and your staff can move these cards to the second list as orders are delivered.
+
+<p align="center">
+	<img src="/images/how-tos/curbside-pickup/trello-lists.png" width="75%">
+</p>
+
+Creating your first board and list in Trello is very straight forward, and well explained in [this official guide](https://trello.com/guide/create-a-board). 
+
+Once you have your lists created, you'll need to collect two bits of data from Trello that are required for the integration with Routegy.
+* Your **Trello API Key** and **Token**
+* **ListId** of your *Waiting Curbside Customers* lists (or however your decided to name it)
```

Comment:
Remove the entire section in parenthesis, it line breaks on the page and isn't needed imo

Reply:
> *Waiting Curbside Customers* lists

It should be singular, "list"